parent/pom.xml

```diff
@@ -1687,5 +1683,24 @@
         </plugins>
       </build>
     </profile>
+
+    <!-- profile to skip flaky test extraction when tests are skipped -->
+    <profile>
+      <id>extract-flaky-tests</id>
```

Comment:
:+1: 

We can use this pattern to enable/disable plugins which don't have a skip property (also for `skipChecks` and so on).

Reply:
It's probably possible, but this might be bit more difficult for the checks, since there already exists a profile to switch from applying the checking rules without failing the build (e.g. format goal of fmt-plugin) to failing build on validation problems (e.g. check goal of fmt-plugin).